Build/libHttpClient..Linux.C/libHttpClient..Linux.C.vcxproj

```diff
@@ -1,197 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
was this file intentionally deleted? Just want to be sure we are not breaking any existing Linux functionality since we are using LHC on Linux for PlayFab Multiplayer

Reply:
it was...confirmed with Sebastian that the linux builds don't use vcxproj files